services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+    return movie_session
+
+
+def get_movies_sessions(session_date: str = None) -> [MovieSession]:
+    movie_session = MovieSession.objects.all()
```

Comment:
Correct the variable name to `movie_sessions`. 
It would be better to filter by `session_date` to simplify the code, but it's not a big deal.